crates/holochain/Cargo.toml

```diff
@@ -85,7 +81,7 @@ strum = "0.18.0"
 subtle-encoding = "0.5"
 tempfile = "3.3"
 thiserror = "1.0.22"
-tokio = { version = "1.36.0", features = ["full"] }
```

Comment:
I expect there are many of these, where we don't actually need all the features of crates enabled. There is a tool I can use to look for this but it takes a long time to run and I need my IDE today... Maybe I should work in a separate clone of Holochain. But either way, not a change I want to put on this PR